extensions/iceberg/build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description 'Iceberg: Support to read iceberg catalogs.'
+
+ext {
+    icebergVersion = '1.4.3'
+    hadoopVersion = '3.3.6'
+}
+
+dependencies {
+    api project(':engine-api')
+    api project(':engine-stringset')
+    api project(':engine-table')
+
+    implementation project(':engine-table')
+    implementation project(':engine-base')
+    implementation project(':log-factory')
+    implementation project(':Configuration')
+    implementation depTrove3
+
+    compileOnly depAnnotations
+
+    Classpaths.inheritAutoService(project)
+    Classpaths.inheritImmutables(project)
+
+    Classpaths.inheritJUnitPlatform(project)
+    Classpaths.inheritAssertJ(project)
+
+    Classpaths.inheritParquetHadoop(project)
+    implementation project(':extensions-parquet-base')
+    implementation project(':extensions-parquet-table')
+    implementation project(':extensions-s3')
+
+    implementation "org.apache.iceberg:iceberg-api:${icebergVersion}"
+    implementation "org.apache.iceberg:iceberg-core:${icebergVersion}"
+    implementation "org.apache.iceberg:iceberg-core:${icebergVersion}:tests"
+    implementation "org.apache.iceberg:iceberg-aws:${icebergVersion}"
+    // implementation "org.apache.iceberg:iceberg-azure:${icebergVersion}"
+    // implementation "org.apache.iceberg:iceberg-gcp:${icebergVersion}"
```

Comment:
You can add a comment explaining when to uncomment these.
 Same for line 49-50.